cmd/command/up/up.go

```diff
@@ -119,3 +119,31 @@ func (p *Plural) handleUp(c *cli.Context) error {
 	utils.Highlight("Feel free to use terraform as you normally would, and leverage the gitops setup we've generated in the apps/ subfolder\n")
 	return nil
 }
+
+func getCluster(cd *cd.Plural) (id string, name string, err error) {
+	if cd == nil {
+		return "", "", fmt.Errorf("please provide a plural client")
```

Comment:
probably should change this error message to something like:

```
You're cli is not logged into Plural, try running `plural login` to generate local credentials.
```

Reply:
done